code/SoloSavingsApp/init-database.sql

```diff
@@ -52,4 +52,14 @@ FOREIGN KEY(user_id) REFERENCES users(user_id) ON DELETE RESTRICT ON UPDATE REST
 -- INSERT STATEMENTS
 -- INSERT INTO `users`(`user_id`, `username`, `email`, `password_hash`, `registration_date`, `balance_amount`, `last_updated`) VALUES ('[value-1]','[value-2]','[value-3]','[value-4]','[value-5]','[value-6]','[value-7]')
 -- INSERT INTO `income`(`income_id`, `user_id`, `source`, `amount`, `income_date`) VALUES ('[value-1]','[value-2]','[value-3]','[value-4]','[value-5]')
--- INSERT INTO `expenses`(`expense_id`, `user_id`, `category`, `amount`, `expense_date`) VALUES ('[value-1]','[value-2]','[value-3]','[value-4]','[value-5]')
\ No newline at end of file
+-- INSERT INTO `expenses`(`expense_id`, `user_id`, `category`, `amount`, `expense_date`) VALUES ('[value-1]','[value-2]','[value-3]','[value-4]','[value-5]')
+
+INSERT INTO `users`(

```

Comment:
Can these changes be echoed in code/database/init-database.sql?

Also I don't think we should have two of these so we would want to remove this one since the database docker file reads from the other.

Reply:
Once these are echoed in the database/init-database.sql this should be good to merge.

Reply:
this was include in the commit by mistake. i'll revert the changes on this file.